operating/examine-images-for-vulnerabilities.adoc

```diff
@@ -45,16 +58,29 @@ It may fail to identify installed components or vulnerabilities if the final ima
 
 |===
 
+include::modules/consolidated-scanner.adoc[leveloffset=+1]
+
+.Additional resources
+
+* For installation instructions for enabling Scanner V4 for {ocp}, see xref:../installing/installing_ocp/install-central-config-options-ocp.adoc#scannerv4-settings_install-central-config-options-ocp[Scanner V4 settings] (Operator) or xref:../installing/installing_ocp/install-central-ocp.adoc#central-services-public-configuration-file-scannerv4_install-central-ocp[Scanner V4] (Helm).
+* For installation instructions for enabling Scanner V4 for Kubernetes by using Helm charts, see xref:../installing/installing_other/install-central-other.adoc#central-services-public-configuration-file-scannerv4_install-central-other[Scanner V4].
+
 include::modules/scanning-images.adoc[leveloffset=+1]
 
+.Additional resources
+
+* For information on the vulnerability sources that the {product-title-short} scanners use, see xref:../architecture/acs-architecture.adoc#vulnerability-sources_acs-architecture[Vulnerability sources].
+
+* For information on integrating with other scanners, see xref:../integration/integrate-with-image-vulnerability-scanners.adoc#integrate-with-image-vulnerability-scanners[Integrating with image vulnerability scanners].
+
```

Comment:
* the `.Additional resources` lines need `[role="_additional-resources"]` on the line immediately above them.
* every addl resource item should be formatted as follows:
```
* xref:../installation/installing-the-product.adoc#installing-the-product[Installing the product]
```
i.e., just an unordered list of simple links without additional text